src/js/react/chat-entry.jsx

```
@@ -0,0 +1,31 @@
+import React, { useState, useEffect } from 'react'
+import { createRoot } from 'react-dom/client'
+import { KapaProvider } from '@kapaai/react-sdk'
+import ChatInterface from './ChatInterface.jsx'
+
+function App() {
```

ESLint

[error] 'App' is defined but never used.

(no-unused-vars)